manifests/init.pp

```diff
@@ -88,6 +88,16 @@
 #                                       mongo database has slow I/O, then setting a higher number may resolve issues where workers are
 #                                       going missing incorrectly.
 #
+# $pulpcore_manage_postgres::           Manage the pulpcore postgresql database
```

Comment:
I am thinking we might as should just expand this out to `postgresql` for references to use the full name

Reply:
@ehelms done